chemprop/models/model.py

```diff
@@ -255,3 +248,22 @@ def load_from_file(cls, model_path, map_location=None, strict=True) -> MPNN:
         model.load_state_dict(state_dict, strict=strict)
 
         return model
+
+
+def load_submodules(checkpoint_path, **kwargs):
```

Comment:
can this be an `@classmethod`? If possible, it seems better because this method is strongly coupled to that function

Reply:
Originally I put it outside `MPNN` to suggest that it also loads submodules for `MulticomponenetMPNN`. But thinking about it again, I agree it should probably be a class method of `MPNN` because then it is more clearly inherited by `MulticomponenetMPNN` when it pulls from `MPNN`. I'll wait to see if Hao-Wei has any comments to address before making that change on Monday.